js/layout-javascript/news-feed-code.js

```diff
@@ -1565,6 +1623,8 @@ DynamicList.prototype.searchData = function(options) {
 
   var _this = this;
   var value = _.isUndefined(options.value) ? _this.searchValue : ('' + options.value).trim();
+  var sortField = _.isUndefined(options.sortField) || '';
```

Comment:
Why have you used `_.isUndefined()` here?

```suggestion
  var sortField = options.sortField || '';
```